org.tuxfamily.StuntRally.yml

```diff
@@ -149,11 +149,19 @@ modules:
       - type: patch
         path: mygui-build.patch
 
-  - name: ENet
+  - name: enet
```

Comment:
Also, please keep the old module name since other modules are also not in lowercase.